src/System.Globalization.Calendars/tests/CalendarHelpers.cs

```diff
@@ -161,49 +164,76 @@ public static IEnumerable<object[]> DateTime_TestData()
         }
 
         [Theory]
-        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year)]
+        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year, false)]
```

Comment:
This test will run for all TFMS including desktop. So why does it pass isDesktopRun=false?